src/project-builder.ts

```diff
@@ -1,11 +1,10 @@
-import { spawnSync } from "node:child_process";
-import { writeFileSync } from "node:fs";
+import { readFileSync, writeFileSync } from "node:fs";
 import { chalk } from "./chalk.js";
 import prompt from "enquirer";
 import exampleConfig from "./tableland.config.example.js";
 
 const docsLink = "https://docs.tableland.xyz";
-const githubLink = "https://github.com/tablelandnetwork";
+const githubLink = "https://github.com/tablelandnetwork/local-tableland";
```

Comment:
I think it makes sense to show the link to this repo instead of the org.  The believe the original link was from when this repo was still private